int/int.mbt

```diff
@@ -48,6 +48,15 @@ pub fn Int::min_value() -> Int {
   min_val
 }
 
```

Comment:
1. Can you add some more documentation here and clarify how you show the correctness of this piece of code?
2. I had another look at xxhash in int64, the while loop is not needed, since it can be unrolled (twice)